db/migrate/20210316083841_add_username_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsernameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :username, :string, null: true
```

Comment:
username should be after uid field

``add_column :users, :username, :string, null: true, after: :uid``